src/bus/p2p.rs

```diff
@@ -169,7 +169,7 @@ impl Reveal {
 #[display("{commit}")]
 pub struct TakerCommit {
     pub commit: Commit,
-    pub public_offer: PublicOffer, // TODO: replace by public offer id
+    pub deal: Deal, // TODO: replace by public deal id
```

Comment:
Comment out-of-date, either
1. we do now the switch to `DealId`
2. update the comment for later
3. we consider that using plain `Deal` is ok, drop the comment

I'd do 1 or 3 personally

Reply:
375046e93ac17e3d1ec54309a1f41d04dacdb0fc